package.json

```diff
@@ -25,8 +25,9 @@
     "prepush": "yarn test && yarn test:routes"
   },
   "dependencies": {
-    "@hmcts/cmc-common-frontend": "^1.17.1",
-    "@hmcts/draft-store-client": "^1.1.1",
+    "@hmcts/cmc-common-frontend": "^1.18.1",
```

Comment:
why don't you update this as well to 1.24.0? (the latest at the moment) 

If you do it later today (or tomorrow) you will also have fix for a11y issues.

Reply:
updated